src/pharmpy/modeling/results.py

```diff
@@ -562,7 +562,8 @@ def calculate_bic(model, type=None):
       | BIC = OFV + n_estimated_parameters * log(n_observations)
     * | random
       | BIC = OFV + n_estimated_parameters * log(n_individals)
-
+    * | iiv
+      | BIC = OFV + n_estimated_iiv_omega_parameters * log(n_individals)
     Parameters
```

Comment:
I think there has to be an empty line before Parameters.

Reply:
Ah, sorry I forgot to keep the empty line! I do it now!